include/eprosimashapesdemo/shapesdemo/ShapePubSubTypes.h

```diff
@@ -20,16 +20,18 @@
  */
 
 
-#ifndef _SHAPE_PUBSUBTYPES_H_
-#define _SHAPE_PUBSUBTYPES_H_
+#ifndef _FAST_DDS_GENERATED_SHAPE_PUBSUBTYPES_H_
+#define _FAST_DDS_GENERATED_SHAPE_PUBSUBTYPES_H_
 
 #include <fastdds/dds/topic/TopicDataType.hpp>
 #include <fastrtps/utils/md5.h>
 
 #include "Shape.h"
 
 #if !defined(GEN_API_VER) || (GEN_API_VER != 1)
-#error Generated Shape is not compatible with current installed Fast DDS. Please, regenerate it with fastddsgen.
+#error \
+
+    Generated Shape is not compatible with current installed Fast DDS. Please, regenerate it with fastddsgen.
```

Comment:
This will fail compilation but not due to the error directive. The blank line should be removed.

Reply:
For what is worth, the result will be the same.

I think it would be a good idea to change this into a `static_assert`.

Reply:
I agree. I'll submit a PR to change this in the Fast DDS-Gen repo and turn it into a static assert.

Reply:
Fixed